### ISSUE TITLE
Is it possible to incorporate background variables into BTM model?

### ISSUE THREAD

**Issue by Evelynhuang** [bnosac/BTM#15]
Hi, 

    Thank you for this great BTM model. I wonder if it is possible to  incorporate background variables into BTM model just like STM model? How to start with? Looking forward to the discussions with you! 

Best, 
Evelyn

**Comment by jwijffels** [comment]
Not that I'm aware of, which paper incorporates BTM with background variables?

**Comment by Evelynhuang** [comment]
Hi,

    There is no research paper for this now. Usually it is done by STM model which can deal with background variables (https://www.structuraltopicmodel.com/).  However, STM only deals with long texts but not short texts. We (a research group from Sweden) would like to use BTM model instead of STM model to incorporate background variables. Do you think it is possible? Can I PM you for more detail?

**Comment by jwijffels** [comment]
If there is no paper which incorporates BTM with background variables, are you looking for someone to invent this and does research to incorporate the STM covariate logic in BTM?

**Comment by Evelynhuang** [comment]
Yes!! We are doing so and would like to cooperate with experts who know BTM model! Hope there is chance to cooperate with you!!

**Comment by jwijffels** [comment]
Ok, feel free to send me an email, you can find my email address at https://cran.r-project.org/web/packages/BTM/index.html
